runtime/browser/lib/x-toast.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+let template = Object.assign(document.createElement('template'), {innerHTML: `
+
+<style>
+  :host {
+    display: block;
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    border: 1px solid silver;
+    transform: translate3d(0, 100%, 0) translateY(-26px);
+    transition: transform 80ms ease-in;
+  }
+  :host([open]) {
+    transform: translate3d(0, 0, 0);
+  }
+  [header] {
+    background-color: white;
+    height: 24px;
+    line-height: 24px;
+    text-align: center;
+    font-size: 0.8em;
+    border-bottom: 1px solid #dddddd;
+    box-sizing: border-box;
+  }
+</style>
+<div header><slot name="header"></slot></div>
+<slot></slot>
+
+`.trim()});
+
+class XToast extends HTMLElement {
```

Comment:
This is worthy of bikeshedding. Basically, if I can't think of a two-word name in under ten seconds, I just use `x-<word>`. 

The `x` comes from `Xenon`, which is a micro-library I've been working on as part of the Polymer project that I've employed now in Arcs. Fwiw, needs of Arcs code has suggested useful refactoring for that library, I'm likely going to name the new stuff `Xen` (there's a mixture now as I explore what works).